tests/server/test_server_route_processing.py

```diff
@@ -1795,3 +1800,65 @@ def test_send_single_get_status_command__gets_processed(
 
     # clean up
     test_process_manager.hard_stop_and_join_processes()
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
```

Comment:
slow parametrized tests are dangerous in their ability to explode the amount of time a test suite takes. are you sure this is necessary to test all 3 conditions in the slow manner (instead of for example confirming with quick unit tests and mocking that all 3 conditions work and then having one slow test as in integration test)?

Reply:
This can definitely just be one slow test